src-chrome-mv3/options.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Popup</title>
+    <style>
+      dt { white-space: nowrap }
+    </style>
+    <script src="options.js"></script>
+  </head>
+  <body>
+    <h1>Imageshare Search</h1>
+    <input type="search" id="search" name="imgs"
+    aria-label="Search for resource matches in Imageshare">
+    <label for="search">Search</label>
+    <button id="standard-search">Run Standard Search</button>
+    <fieldset>
+      <dl>
+        <dt><label for="subject">Subject: </label></dt>
+        <dd><select id="subject" enabled>
+        </select></dd>
+
+        <dt><label for="type">Type: </label></dt>
+        <dd><select id="type" enabled>
+        </select></dd>
+
+        <dt><label for="accommodation">Accommodation: </label></dt>
+        <dd><select id="accommodation" enabled>
+        </select></dd>
+
+        <dt><label for="source">Source: </label></dt>
+        <dd><select id="source" enabled>
+        </select></dd>
+
+      </dl>
+    </fieldset>
+    <button id="advanced-search">Advanced Search</button>
+
+    <p>Would you like to save these settings?</p>
+    <div>
+      <input type="checkbox" id="save-advanced" name="save-advanced"
+             unchecked>
```

Comment:
`unchecked` is not a valid attribute (`checked` is) - the default state is unchecked.

Reply:
change enacted in alerts-cross-browser